src/gridded/indexing.jl

```diff
@@ -86,3 +86,11 @@ function getindex{T,N,TCoefs,IT<:DimSpec{Gridded},K,P}(itp::GriddedInterpolation
     dest = Array(T, map(length, x))::Array{T,N}
     getindex!(dest, itp, x...)
 end
+
+function getindex_return_type{T,N,TCoefs,IT<:DimSpec{Gridded},K,P}(::Type{GriddedInterpolation{T,N,TCoefs,IT,K,P}}, argtypes)
+    Tret = TCoefs
+    for a in argtypes
+        Tret = Base.promote_op(Base.MulFun, T, a)
```

Comment:
Is this loop body intended to make use of `Tret = TCoefs` above somehow, or is that just a default in case `length(argtypes) == 0`?